dist/index.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+  <link rel="stylesheet" href="./style.css">
+  <title>Restaurant page</title>
+</head>
+
+<body>
+  <div class="container">
+    <input class="input-radio" type="radio" name="tabs" checked id="tab-1">
+    <label class="tab-label" for="tab-1">Home</label>
+    <input class="input-radio" type="radio" name="tabs" id="tab-2">
+    <label class="tab-label" for="tab-2">Menu</label>
+    <input class="input-radio" type="radio" name="tabs" id="tab-3">
+    <label class="tab-label" for="tab-3">Contact</label>
```

Comment:
`body` should only contain a `div` with `id=content` and all elements should be appended to it using JavaScript ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)).